src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        final int amountFigures = 6;
```

Comment:
The name `amountFigures` could be more descriptive and consistent with common naming conventions. Consider renaming it to something like `FIGURES_COUNT` to indicate that it's a constant representing the count of figures.